### ISSUE TITLE
The pane will not display in Bing tabs

### ISSUE THREAD

**Issue by AmineI** [XFox111/TabsAsideExtension#58]
### Description
On Edge, When doing a search on bing.com, the pane won't open. After some investigation on my side, I discovered that when you create an iframe on bing, from the console or a script, it is instantly removed by an obfuscated script from bing.

According to what I could decipher from the bing script, it is potentially to prevent search hijacking or malicious ads (some malware extensions hijack the bing page, to replace the bing results by spam results, and many users recently started complaining about that in edge forums. It could explain such a change on bing's side.)

### Reproduction steps
1. Go to https://www.bing.com/search?q=A+search . 
2. Click on the tabs extension icon

### Expected behavior
The tabs aside pane should have opened. But it did not.
(Note that Bing seems to like controlled rollouts, where they deploy features to a subset of users at first, so it is possible that you don't see this behavior yet)

### Environment
Please provide the following information:
 - Operating System: Windows 10 Pro 20H2
 - Browser: Microsoft Edge Stable 87.0.664.41
 - Extension version: All versions using an iframe. That includes last stable one, 1.9.1, and the next one, 2.0 . That should be unrelated to the extension, because trying to create an iframe from the DevTools console exhibits the same behavior

**Comment by AmineI** [comment]
I've been thinking about what to do, as it could be related to one of the recent chrome webstore reviews of the extension, where an user says the tabs won't open anymore on his end. 

Perhaps a correct workaround would be that if the pane somehow won't open in an iframe (for any reason), we open it in a full page ?

**Comment by XFox111** [comment]
Hmm... I dunno, seems fine to me. Edge and Windows versions are the same. Are you sure, it fails because of Bing or maybe it's just other extensions?
![image](https://user-images.githubusercontent.com/28831743/100141743-ed41ee00-2ea3-11eb-8067-5ec650de6fdf.png)

**Comment by AmineI** [comment]
> Hmm... I dunno, seems fine to me. Edge and Windows versions are the same. Are you sure, it fails because of Bing or maybe it's just other extensions?

100% sure it's from bing - I retried on a fresh browser profile with only the store version of this extension, and the code deleting added elements is straight from Bing 
![image](https://user-images.githubusercontent.com/15179425/100143727-093e9300-2e96-11eb-9836-dc1fdbda5cd8.png)

I see on your screenshot that we have different Bing interfaces as well so they are probably still rolling out the change.. I'll be sure to check regularly and update this.

**Comment by nvmkpk** [comment]
I ran into this with my extension. To work around it, I set the initial width and height of the iframe is 0 and make it larger little later using setTimeout.

**Comment by XFox111** [comment]
This is super weird. I keep getting zero issues on this 🤔

**Comment by nvmkpk** [comment]
The reason may be that bing does it differently for different regions.

**Comment by XFox111** [comment]
@AmineI, @nvmkpk, What's your regions? Mine is Russia

**Comment by nvmkpk** [comment]
I am in US.

**Comment by XFox111** [comment]
@nvmkpk yeah, that makes sense. I've just tried to reproduce the bug with turned on US VPN and InPrivate mode and it does block the pop-up

**Comment by XFox111** [comment]
Unfortunately, I have no time to take the issue, so I'd appreciate if someone could fix it
